src/app/config/health-checks/new-health-check/new-health-checks.component.ts

```diff
@@ -1,61 +1,152 @@
 import { FormControl, FormGroup, Validators } from '@angular/forms';
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 
+import {
+  CRACKER_TYPE_FIELD_MAPPING,
+  CRACKER_VERSION_FIELD_MAPPING
+} from 'src/app/core/_constants/select.config';
+import { attack, hashtype } from 'src/app/core/_constants/healthchecks.config';
 import { AlertService } from 'src/app/core/_services/shared/alert.service';
 import { GlobalService } from 'src/app/core/_services/main.service';
 import { PageTitle } from 'src/app/core/_decorators/autotitle';
 import { SERV } from '../../../core/_services/main.config';
+import { transformSelectOptions } from 'src/app/shared/utils/forms';
+import { UnsubscribeService } from 'src/app/core/_services/unsubscribe.service';
+import { AutoTitleService } from 'src/app/core/_services/shared/autotitle.service';
 
 @Component({
   selector: 'app-new-health-checks',
   templateUrl: './new-health-checks.component.html'
 })
-@PageTitle(['New Health Check'])
-export class NewHealthChecksComponent implements OnInit {
-
-  // Form create Health Check
+export class NewHealthChecksComponent implements OnInit, OnDestroy {
+  /** Form group for Health Checks */
   createForm: FormGroup;
```

Comment:
I don't really like the prefix `create` on a variable name. It suggests that it is an action to me. How about just `form` or `healthChecksForm`?

Reply:
I have been using form in other components, makes more sense. I'll change it.